frontend/src/components/Search.tsx

```
@@ -86,9 +88,11 @@
             />
             {searchQuery && (
               <button
-                className="absolute top-1/2 right-2 -translate-y-1/2 rounded-full p-1 hover:bg-gray-100 focus:ring-2 focus:ring-gray-300 focus:outline-hidden"
+                type="button"
+                aria-label="Clear search"
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)